### PR TITLE
Update for MeSH Autocomplete

### PR DESCRIPTION
**Pull request by alliomeria**
For https://github.com/esmero/ami/issues/185, to complement  https://github.com/esmero/ami/pull/186

Update logic for processing MeSH queries:
- for count > 0 (instead of 1)
- make use of array_unshift to better sort and select results that more closely match original query

Coded live by @DiegoPino and shared with @alliomeria for local testing. Thank you @DiegoPino!

### CODE REVIEW

File: src/Controller/AuthAutocompleteController.php [hunk 1]
```diff
@@ -916,24 +916,31 @@ protected function mesh($input, $vocab, $rdftype) {
...
-    $json_error = json_last_error();
-    if ($json_error == JSON_ERROR_NONE) {
-      if (count($jsondata) > 1) {
-        foreach ($jsondata as $entry) {
-          $results[] = [
-            'value' => $entry['resource'],
-            'label' => $entry['label'],
-          ];
-        }
+$input_encoded = rawurlencode($input);
```
Comment (DiegoPino) [inline src/Controller/AuthAutocompleteController.php:919]: @alliomeria bit like the last one, could you in this case restore the indentation? Basically move everything right ?

Comment (alliomeria) [reply src/Controller/AuthAutocompleteController.php]: How's this now @DiegoPino ?

**Review by DiegoPino — APPROVED** [review]
Amazing! thanks so much

**Comment by alliomeria** [comment]
Thank you for your review & feedback @DiegoPino !

**Pull request merged**